proposal/js-deprecations.d.ts

```diff
@@ -109,35 +117,54 @@ declare module '../spec/js-api' {
     /**
      * A set of deprecations to treat as fatal.
      *
-     * If a version is provided, then all deprecations released in that
-     * version or earlier will be treated as fatal. If a string is provided, it
-     * will resolve to the deprecation with that ID, or error if none exists.
+     * If a deprecation warning of any provided type is encountered during
+     * compilation, the compiler must error instead.
+     *
+     * The compiler should convert any string passed here to a `Deprecation`
+     * by indexing `Deprecations`.
```

Comment:
Nit:

```suggestion
     * by indexing `deprecations`.
```

...since this is the actual field with concrete values.

Reply:
Done